chromium_src/components/infobars/core/infobar_delegate.h

```diff
@@ -14,14 +14,16 @@
 
 #define BRAVE_INFOBAR_DELEGATE_IDENTIFIERS                                     \
   BRAVE_CONFIRM_P3A_INFOBAR_DELEGATE = 500,                                    \
-  WAYBACK_MACHINE_INFOBAR_DELEGATE = 502,                                      \
-  SYNC_V2_MIGRATE_INFOBAR_DELEGATE = 503,                                      \
-  ANDROID_SYSTEM_SYNC_DISABLED_INFOBAR = 504, SYNC_CANNOT_RUN_INFOBAR = 505,   \
+  WAYBACK_MACHINE_INFOBAR_DELEGATE = 502, SYNC_CANNOT_RUN_INFOBAR = 505,       \
   WEB_DISCOVERY_INFOBAR_DELEGATE = 506,                                        \
   BRAVE_SYNC_ACCOUNT_DELETED_INFOBAR = 507,                                    \
   BRAVE_REQUEST_OTR_INFOBAR_DELEGATE = 508, BRAVE_IPFS_INFOBAR_DELEGATE = 509, \
   DEV_CHANNEL_DEPRECATION_INFOBAR_DELEGATE = 510,
 
+// Deprecated:
+// SYNC_V2_MIGRATE_INFOBAR_DELEGATE = 503
```

Comment:
Should this value also be commented out in brave/android/java/org/chromium/chrome/browser/infobar/BraveInfoBarIdentifier.java?

Reply:
Yes, it should be commented out there also.
Here is that place:
https://github.com/brave/brave-core/pull/20906/files#diff-904cb9b3f638eaabd683bab4bc469c0ac20dcce6f6a30afca30b83be10f660c6R26